### DISCUSSION TITLE
Search input field narrow when using long repository name

### DISCUSSION THREAD

**Discussion by MateuszGrabuszynski** [community/community#35834] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,
I added `cs.` to address of the repository I use. Observed, that when the repository has a long name (77 characters, organization name + repository name), then the querying is really hard to perform. The search field becomes very narrow.
![image](https://user-images.githubusercontent.com/22837924/195214483-7dc013fe-b083-497c-9123-88663eb44e3d.png)

**Comment by Akash1134** [comment]
Hi @MateuszGrabuszynski , thanks for reporting the problem. Please refer to this [discussion](https://github.com/orgs/community/discussions/36514) where one of our engineers has addressed this issue.

**Comment by MateuszGrabuszynski** [comment]
Hey @Akash1134, thank you for your answer but I think the discussion you mentioned is more about the backend (filtering and sorting being outdated), and my concern is about the frontend (how the field looks on `cs.github.com/full_repo_name`). Moreover, the link goes to a topic that is not cs-related, and the point from my comment is still valid, the field is even narrower for a fork (see screenshot).
![image](https://user-images.githubusercontent.com/22837924/199223213-b20d6526-bc4e-4141-917c-e0e839f6db17.png)
Could you please take another look?

**Comment by look** [comment, marked as answer]
@MateuszGrabuszynski thanks for the report. Fortunately, we should have a work around for this soon.
